### ISSUE TITLE
Can't authenticate on device running API 19

### ISSUE THREAD

**Issue by wolkenschieber** [spotify/android-auth#11]
## Description

Trying to authenticate to Spotify on a decive running API 19, an empty authentication result is returned.

### Versions
Android SDK version: `19`
Spotify app version: `com.spotify.sdk:spotify-android-auth-1.0.0@aar`
Android OS version `4.4`

## Steps to Reproduce
1. Setup API 19 AVD decive. I used *API 19, Intel x86 Atom System Image*
2. Authenticate to Spotify using `auth-sample` from [spotify/android-auth.git](https://github.com/spotify/android-auth.git)

## Logs

```
E/com.spotify.sdk.android.authentication.LoginActivity: Can't use LoginActivity with a null caller. Possible reasons: calling activity has a singleInstance mode or LoginActivity is in a singleInstance/singleTask mode
```

### `auth-lib` modification

Poking around the `auth-lib`-code, I saw that the manifest defines `android:launchMode="singleInstance"`. If I remove this line, the sample app receives a token, but crashes immediatelly afterwards.

Using this this library with `standard` launchMode seems to work in my application. But due to the behaviour in the sample app, I'm unsure about the consequences.

### Overriding launch mode in `LoginActivity`

If I override the LoginActivity's launch mode using:

```
        <activity
            tools:replace="android:launchMode"
            android:launchMode="standard"
            android:name="com.spotify.sdk.android.authentication.LoginActivity"
            android:theme="@android:style/Theme.Translucent.NoTitleBar">
        </activity>
```

API 19 devices are able to login without further library changes, however API 25 is acting weird then

**Comment by wolkenschieber** [comment]
Test with integrated fix of [Fix authentication issue on API 19][https://github.com/spotify/android-auth/pull/22]

**Comment by erikghonyan** [comment]
Fixed in #22
